### PR TITLE
Split org Propfile README to a new tab `overview`

### PR DESCRIPTION
**Pull request by a1012112796**
like user profile, add a new overviw tab to show profile READEME when it is exist.

replace #31349 (another solution option)

example view:
屏幕截图 2024-06-14 094116
屏幕截图 2024-06-14 094101

### CODE REVIEW

**Comment by lunny** [comment]
Looks like Github's default page is repositories list.

**Comment by silverwind** [comment]
Github renders:

- Overview tab: Readme + Repos
- Repos tab: Repos

I suggest we do the same, which should make everyone happy.

**Comment by silverwind** [comment]
> Looks like Github's default page is repositories list.

No, if I open https://github.com/go-gitea, it goes to the the overview tab.

**Comment by a1012112796** [comment]
> Github renders:
> 
> * Overview tab: Readme + Repos
> * Repos tab: Repos
> 
> I suggest we do the same, which should make everyone happy.

has add it https://github.com/go-gitea/gitea/pull/31373/commits/990584f9bd5734478f8e8de8bf9f8a66ae149864

File: templates/org/menu.tmpl [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <div class="ui container">
 	<overflow-menu class="ui secondary pointing tabular borderless menu tw-mb-4">
 		<div class="overflow-menu-items">
-			<a class="{{if .PageIsViewRepositories}}active {{end}}item" href="{{$.Org.HomeLink}}">
+			{{if .HasProfileReadme}}
+				<a class="{{if .PageIsViewOverview}}active {{end}}item" href="{{$.Org.HomeLink}}">
+					{{svg "octicon-info"}} {{ctx.Locale.Tr "user.overview"}}
+				</a>
+			{{end}}
+			<a class="{{if .PageIsViewRepositories}}active {{end}}item" href="{{$.Org.HomeLink}}{{if .HasProfileReadme}}?view_repositorys=true{{end}}">
```
Comment (silverwind) [inline templates/org/menu.tmpl]: Can you make this route `/<orgname>/repositories` please? Query string is ugly and your `view_repositorys` is actually a typo.

**Review by techknowlogick — APPROVED** [review]

**Review by wolfogre — APPROVED** [review]

**Comment by lunny** [comment]
This is not ideal because I think the members and team on the right sidebar of repository list can be removed since there is an overview tab. Of course, it can be finished by another PR.

**Comment by wolfogre** [comment]
This PR has been in `reviewed/wait-merge` status for long enough. I think it's time to merge.

**Pull request merged**